SaintCoinach/Definitions/Materia.json

```diff
@@ -4,7 +4,7 @@
   "definitions": [
     {
       "type": "repeat",
-      "count": 10,
+      "count": 16,
```

Comment:
While I understand your reasoning behind that, it's full of 0 atm and if there's a rule we are following on definitions, it's to never assume what will be added in the future. So assuming these columns are for 6 more materia grades is not the way to go until we have a proof for that.

Reply:
ffxiv_dx11.exe ver. 2023.07.26.0000.0000: 140988E10
![image](https://github.com/xivapi/SaintCoinach/assets/64143453/798a1bcd-b8a6-4c9b-af40-abc118b3497a)
a size 16 iterator is used